apps/vx-scan/frontend/src/api.ts

```diff
@@ -14,3 +21,42 @@ export function useApiClient(): grout.Client<Api> {
   }
   return apiClient;
 }
+
+export const queryClientDefaultOptions: DefaultOptions = {
+  queries: {
+    // Since our backend is always local, we don't want react-query to "pause"
+    // when it can't detect a network connection.
+    networkMode: 'always',
+
+    // Never mark cached data as stale automatically. This will prevent
+    // automatic refetching of data (e.g. upon navigating to a page). Cached
+    // queries will only be refecthed when we explicitly invalidate the query
+    // after a mutation. This is an appropriate strategy in VxSuite since
+    // there is only ever one frontend client updating the backend, so we
+    // don't expect data to change on the backend except when we mutate it.
+    staleTime: Infinity,
+  },
+  mutations: { networkMode: 'always' },
+};
+
+export const getConfig = {
+  queryKey(): QueryKey {
+    return ['getConfig'];
+  },
```

Comment:
Cool! I like this pattern for keeping all data for a query colocated while also making it easy to cross-reference query keys in other queries

I take it `queryKey` is a function because, for some queries, query keys will contain IDs, etc. that need to be passed in?